### PR TITLE
WRO-9305: Enact Browser background color is white when silicon night skin

### PR DESCRIPTION
**Pull request by alexandrumorariu**
Enact-DCO-1.0-Signed-off-by: Alexandru Morariu alexandru.morariu@lgepartner.com

### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [X] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [X] Documentation was verified or is not changed
* [X] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Changed where the background-color and backgroung-image are applied in ThemeDecorator.module.less to mirror the Sandstone changes and apply them properly.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Because of the applied page background color and image the screenshot tests failed and there are differences in the page background of storybook. However there are no layout shifts and no change in the color of the components themselves.
This effect is present in sandstone also with neutral and light skins.

### Links
[//]: # (Related issues, references)
WRO-9305

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/MediaPlayer` layout issues for Cobalt, Carbon, Copper, Electro, Titanium skins
 - `agate/Picker` and `agate/RangePicker` to match latest design for Silicon skin
 - `agate/Scroller` to update scrollButtons state on initial render
+- `agate/ThemeDecorator` fixed a problem with background color and image
```
Comment (daniel-stoian-lgp) [inline CHANGELOG.md:13]:
```suggestion
- `agate/ThemeDecorator` so `bg` class works only at the apps' root node not at a child node of the app's root node
```

**Review by daniel-stoian-lgp — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/MediaPlayer` layout issues for Cobalt, Carbon, Copper, Electro, Titanium skins
 - `agate/Picker` and `agate/RangePicker` to match latest design for Silicon skin
 - `agate/Scroller` to update scrollButtons state on initial render
+- `agate/ThemeDecorator` so `bg` class works only at the apps' root node not at a child node of the app's root node
```
Comment (seunghoh) [inline CHANGELOG.md:13]: Would you please rewrite the changelog to the same format as other Fixed section? 
I'd try to read but I can't understand.
(Fixed) `agate/ThemeDecorator` so `bg` class works only at the... ? 
-> (Fixed) `agate/ThemeDecorator` to ACTION...

**Review by seunghoh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (alexandrumorariu) [reply CHANGELOG.md]: I have pushed a fix for the CHANGELOG. Please let me know if it's better now.

File: ThemeDecorator/ThemeDecorator.module.less [hunk 3]
```diff
@@ -72,17 +72,20 @@
...
-			background-image: @agate-bg-image;
-
-			&::before {
-				content: "";
-				display: block;
-				background-image: @agate-bg-image2;
-				position: absolute;
-				.position(0);
-			}
+		&.bg ::before {
```
Comment (0x64) [inline ThemeDecorator/ThemeDecorator.module.less:75]: We usually don't put a space for pseudo classes and pseudo elements. Would you change this?

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/MediaPlayer` layout issues for Cobalt, Carbon, Copper, Electro, Titanium skins
 - `agate/Picker` and `agate/RangePicker` to match latest design for Silicon skin
 - `agate/Scroller` to update scrollButtons state on initial render
+- `agate/ThemeDecorator` to properly apply the background color and background image
```
Comment (0x64) [inline CHANGELOG.md:13]:
```suggestion
- `agate/ThemeDecorator` to apply the background color and background image properly
```

**Review by seunghoh — APPROVED** [review]

**Pull request merged**